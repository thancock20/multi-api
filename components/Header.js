const Header = () => (
  <header>
    <h1>Multi-API</h1>
  </header>
)

export default Header
