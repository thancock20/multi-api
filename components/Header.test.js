import Header from './Header';

describe('Header component', () => {
  it('renders with a title', () => {
    const wrapper = shallow(<Header title="This is the title" />);
    expect(wrapper).toMatchSnapshot();
  });
});
