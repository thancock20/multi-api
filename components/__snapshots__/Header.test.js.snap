// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component renders with a title 1`] = `
<header
  className="jsx-1919171108"
>
  <h1
    className="jsx-1919171108"
  >
    This is the title
  </h1>
  <JSXStyle
    css="header.jsx-1919171108{text-align:center;}h1.jsx-1919171108{color:inherit;font-size:2.5rem;font-family:'Source Serif Pro',serif;}"
    styleId="1919171108"
  />
</header>
`;
